deeplake/integrations/mmdet/mmdet_.py

```diff
@@ -0,0 +1,1027 @@
+            if cls < len(self.CLASSES):
+                row_data += [f"{cls} [{self.CLASSES[cls]}]", f"{count}"]
+            else:
+                # add the background number
+                row_data += ["-1 background", f"{count}"]
+            if len(row_data) == 10:
+                table_data.append(row_data)
+                row_data = []
+        if len(row_data) >= 2:
+            if row_data[-1] == "0":
+                row_data = row_data[:-2]
+            if len(row_data) >= 2:
+                table_data.append([])
+                table_data.append(row_data)
+
+        table = AsciiTable(table_data)
+        result += table.table
+        return result
+
+    def format_results(self, results, jsonfile_prefix=None, **kwargs):
+        """Format the results to json (standard format for COCO evaluation).
+
+        Args:
+            results (list[tuple | numpy.ndarray]): Testing results of the
+                dataset.
+            jsonfile_prefix (str | None): The prefix of json files. It includes
+                the file path and the prefix of filename, e.g., "a/b/prefix".
+                If not specified, a temp file will be created. Default: None.
+
+        Returns:
+            tuple: (result_files, tmp_dir), result_files is a dict containing
+                the json filepaths, tmp_dir is the temporal directory created
+                for saving json files when jsonfile_prefix is not specified.
+        """
+        assert isinstance(results, list), "results must be a list"
+        assert len(results) == len(
+            self
+        ), "The length of results is not equal to the dataset len: {} != {}".format(
+            len(results), len(self)
+        )
+
+        if jsonfile_prefix is None:
+            tmp_dir = tempfile.TemporaryDirectory()
+            jsonfile_prefix = osp.join(tmp_dir.name, "results")
+        else:
+            tmp_dir = None
+        result_files = self.results2json(results, jsonfile_prefix)
+        return result_files, tmp_dir
+
+
+def load_ds_from_cfg(cfg: Dict):
+    creds = cfg.get("deeplake_credentials", {})
+    token = creds.get("token", None)
+    if token is None:
+        uname = creds.get("username")
+        if uname is not None:
+            pword = creds["password"]
+            client = DeepLakeBackendClient()
+            token = client.request_auth_token(username=uname, password=pword)
+    ds_path = cfg.deeplake_path
+    ds = dp.load(ds_path, token=token, read_only=True)
+    deeplake_commit = cfg.get("deeplake_commit")
+    deeplake_view_id = cfg.get("deeplake_view_id")
+    deeplake_query = cfg.get("deeplake_query")
+
+    if deeplake_view_id and deeplake_query:
+        raise Exception("A query and view_id were specified simultaneously for a dataset in the config. Please specify either the deeplake_query or the deeplake_view_id.")
+    
+    if deeplake_commit:
+        ds.checkout(deeplake_commit)
+
+    if deeplake_view_id:
+        ds.load_view(id=deeplake_view_id)
```

Comment:
@FayazRahman @farizrahman4u @adolkhan Where is the resulting view being passed? I thing you're loading the view and not doing anything with it.